src/components/submission/feed-entry.vue

```diff
@@ -173,9 +200,19 @@ export default {
       text-align: center;
       width: 18px;
     }
+
+    .submission-feed-entry-entity-data {
+      font-weight: normal;
+      color: $color-action-foreground;
```

Comment:
I think this color in the release criteria is aspirational linking. Since things won't link yet, probably no need to color it like a link yet.